src/net/udns/udns_resolver.c

```diff
@@ -887,7 +887,7 @@ dns_submit_dn(struct dns_ctx *ctx,
   SETCTXOPEN(ctx);
   dns_assert_ctx(ctx);
 
-  q = (struct dns_query *)calloc(sizeof(*q), 1);
+  q = (struct dns_query *)calloc(1, sizeof(*q));
```

Comment:
This should probably also be fixed in the upstream udns code.

Reply:
oh weird that this isn't available as an external dependency.